zabbix/stackstate_checks/zabbix/zabbix.py

```diff
@@ -196,9 +194,6 @@ def process_host_topology(self, topology_instance, zabbix_host, stackstate_envir
         identifiers = list()
         # get actual hostname from agent
         # if zabbix is deployed on ec2 instance, it doesn't give any metadata about actual hostname
-        hostname = self.get_hostname()
-        identifiers.append(Identifiers.create_host_identifier(hostname))
-        identifiers.append(hostname)
         identifiers.append(zabbix_host.host)
 
```

Comment:
Create an identifier on the pattern `urn:host:/<host>`. you can use this 200 line copy.